desktop/renderer-app/src/pages/LoginPage/googleLogin.ts

```diff
@@ -0,0 +1,53 @@
+import { setAuthUUID, loginProcess } from "../../api-middleware/flatServer";
+import { v4 as uuidv4 } from "uuid";
+import { LoginExecutor } from "./utils";
+import { shell } from "electron";
+import { errorTips } from "../../components/Tips/ErrorTips";
+import { FLAT_SERVER_LOGIN } from "../../api-middleware/flatServer/constants";
+import { GOOGLE } from "../../constants/process";
+
+// @TODO: migrate to new google login api before 2023
```

Comment:
```suggestion
// @TODO: migrate to new google login api before 2023. @hyrious 
```

It should point to a specific member.